source/gui/settingsDialogs.py

```diff
@@ -3295,6 +3295,23 @@ def __init__(self, parent):
 
 		self.Layout()
 
+		silenceDurationLabelText = _(

```

Comment:
There is an "audio" grouping in the advanced settings panel. Could you add this parameter there rather than at the end? And update the documentation the same way?